lib/strerror.c

```diff
@@ -661,28 +662,19 @@ get_winapi_error(int err, char *buf, size_t buflen)
 
   *buf = '\0';
 
-#ifdef _WIN32_WCE
```

Comment:
There didn't seem to be any meaningful difference between the Windows CE version of this function and the regular Windows version, so I merged them.